Chap_API_Struct.tex

```diff
@@ -1431,6 +1487,78 @@ \subsubsection{Info structure list macros}
 
 Copy the provided key and data into a \refstruct{pmix_info_t} on the list. The key and any data stored in the source value can be modified or free'd without affecting the copied data once the function has completed.
 
+%%%%%%%%%%%
+\littleheader{Retrieve next \refstruct{pmix_info_t} on a list}
+\declareapiProvisional{PMIx_Info_list_get_info}
+
+%%%%
+\summary
+
+Retrieve next \refstruct{pmix_info_t} on a list.
+
+%%%%
+\format
+
+\versionMarker{5.0}
+\cspecificstart
+\begin{codepar}
+pmix_info_t *
+PMIx_Info_list_get_info(void *ptr,
```

Comment:
```suggestion
PMIx_Info_list_get_info(void *head,
```

Reply:
While I think this is a good suggestion, the rest of the `PMIx_Info_list_*` APIs use the `ptr` name for the list being passed in. If we decide to rename the parameter, we should do all the list APIs at once, IMO.

Reply:
Agreed